tests/utils/mock_microgrid.py

```diff
@@ -71,16 +74,21 @@ def __init__(self, components: Set[Component], connections: Set[Connection]):
             id: channel.new_sender() for id, channel in ev_charger_channels.items()
         }
 
-    def initialize(self, mocker: MockerFixture) -> None:
+    def initialize(self, mocker: MockerFixture | None) -> None:
         """Mock `microgrid.get` call to return this mock_microgrid.
 
         Args:
-            mocker: mocker from the current test
+            mocker: mocker from the current test or `None` to use default mocker.
         """
+        params = ("frequenz.sdk.microgrid._microgrid._MICROGRID", self.mock_microgrid)
```

Comment:
Now that there's only one call to `mocker.patch`,  I guess you don't need to separate the args from the call anymore.